RELEASE_NOTES.md

```diff
@@ -1,42 +1,279 @@
+
+### Breaking Changes
+
+#### AIX Virtualization Improvements
+
+The Ohai :Virtualization plugin on AIX systems will now properly return the `lpar_no` and `wpar_no` values as Integers instead of Strings. This makes the data much easier to work with in cookbooks, but may be a breaking change depending on how AIX users consumed these values.
+
+#### 32bit RHEL/CentOS 6 Support
+
+Chef Infra Client 17 packages will no longer be produced for 32bit RHEL/CentOS 6 systems. RHEL/CentOS 6 became EOL November 2020. We are extending support for 64 bit RHEL/CentOS 6 until Chef Infra Client 18 (April 2022) or when upstream platform or library changes prohibit us from building on these EOL systems.
+
+#### Chef Client As A Service on Windows
+
+Based on customer feedback and observations in the field we've removed the ability to run the Chef Infra Client as a service on Windows nodes. We've seen the service manager for the Chef Infra Client consume excessive memory, hang preventing runs, or prevent nodes from updating to new client releases properly. We've always seen significantly better reliability by running Chef Infra Client as a scheduled task on Windows and in July of 2006 we introduced warnings to the [chef-client cookbook](https://supermarket.chef.io/cookbooks/chef-client) when running as a service. The ability to setup the client as a service was later removed from the cookbook entirely in October of 2017.
+
+For customers current running Chef Infra Client as a service we suggest migrating to scheduled task based execution. This allows for complex scheduling scenarios not possible with simple services such, as the ability to skip Chef Infra Client execution if a system is on battery or to run the Chef Infra Client as soon as a system boots to ensure configuration.
+
+Chef Infra Client can be configured to run as a scheduled task using the [chef-client cookbook](https://supermarket.chef.io/cookbooks/chef-client) or ideally using the [chef_client_scheduled_task resource](https://docs.chef.io/resources/chef_client_scheduled_task/) built into Chef Infra Client 16 or later. For users already running as a service setting up the scheduled task and then stopping the existing service can be performed within a Chef Infra Client run to migrate systems.
+
+#### Gem Resource Ruby 1.9+
+
+The `gem` resource used to install Ruby Gems into the system's Ruby installation will now assume Ruby 1.9 or later. As Ruby 1.8 or below have been end of life for nearly 7 years we believe there is little to no impact in this change.
+
+#### Legacy node['filesystem2'] removed on AIX/Solaris/FreeBSD
+
+The legacy `node['filesystem2']` attributes leftover from our multi-year migration of filesystem data on AIX, Solaris, and FreeBSD systems has been removed. This same data is now available at `node['filesystem']`
+
```

Comment:
Given that https://github.com/chef/ohai/pull/1662 is now merged, we should probably amend this like so...

```suggestion
#### Legacy node['filesystem2'] removed on AIX/Solaris/FreeBSD

The legacy `node['filesystem2']` attributes leftover from our multi-year migration of filesystem data on AIX, Solaris, and FreeBSD systems has been removed. This same data is now available at `node['filesystem']`

#### node['filesystem'] on Windows updated to new-style data

`node['filesystem']` on Windows now has what `node['filesytem2']` has had on Windows throughout Chef 16. Both node attributes now have the same data as a way to allow users to transition from `filesystem2` to `filesystem` safely. In Chef 18, we will remove `node['filesystem2']` completely thus finishing our multi-year migration of Ohai filesystem data format.
```